webapp/migrations/Version20221004135409.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+declare(strict_types=1);
+
+namespace DoctrineMigrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+final class Version20221004135409 extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return 'Allow super lazy evaluation where nothing is judged unless requested.';
+    }
+
+    public function up(Schema $schema): void
+    {
+        $this->addSql('ALTER TABLE contestproblem CHANGE lazy_eval_results lazy_eval_results INT UNSIGNED DEFAULT NULL COMMENT \'Whether to do lazy evaluation for this problem; if set this overrides the global configuration setting\'');
+        $this->addSql('UPDATE contestproblem SET lazy_eval_results=2 WHERE lazy_eval_results=0');
```

Comment:
do we have to update configuration as well if it exists?

Reply:
Probably yes, I'll check.